awx/ui_next/src/util/validators.js

```diff
@@ -57,6 +57,12 @@ export function minLength(min) {
 
 export function minMaxValue(min, max) {
   return (value) => {
+    if (!Number.isFinite(min) && value > max) {
+      return t`This field must be a number and have a value less than ${max}`;
+    }
+    if (!Number.isFinite(max) && value < min) {
+      return t`This field must be a number and have a value greater than ${min}`;
```

Comment:
This is how the ${min} is being displayed.
 
![image](https://user-images.githubusercontent.com/12464858/127211211-4a153f9b-ce8c-47bc-8600-27d3c885de25.png)